packages/base/package.json

```diff
@@ -21,17 +25,29 @@
   "scripts": {
     "build": "npm run build:types && npm run build:js",
     "build:types": "tsc --declaration --declarationMap --emitDeclarationOnly",
-    "build:js": "babel --root-mode upward src --out-dir lib --extensions .ts -s",
+    "build:js": "npm run build:old && babel --root-mode upward src/blockchain.ts --out-file lib/blockchain.js --source-maps",
+    "build:old": "mkdir -p lib && cp -rp src/*.d.ts lib && cp -rp src/*.js lib",
```

Comment:
is the copy step to be must here? can we replace with tsc's `allowJs`?

Reply:
Using `allowJs` will generate `xxx.d.ts` also, which is not what we want.